build_integration.sh

```diff
@@ -54,6 +54,9 @@ pushd servers
           fi
           echo
           echo "Extracting kafka ${kafka} binaries"
+          if [ ! -d ../$kafka ]; then
+            mkdir ../$kafka
+          fi
```

Comment:
Like above, this feels like a distinct PR / commit which wouldn't happen if I merge by squashing